versions.mk

```diff
@@ -10,7 +10,8 @@ K8S_VER ?= 1.21.2
 # major + minor padded to 2 chars with 0 + patch also padded to 2 chars, e.g.
 # 1.13.5 -> 11305, 1.13.12 -> 11312, 2.0.0 -> 20000 and so on
 K8S_VER_SUFFIX ?= $(shell printf "%d%02d%02d" $(shell echo $(K8S_VER) | sed "s/\./ /g"))
-PLANET_TAG ?= 9.0.3-$(K8S_VER_SUFFIX)
+#PLANET_TAG ?= 9.0.4-$(K8S_VER_SUFFIX)
+PLANET_TAG ?= 9.0.3-$(K8S_VER_SUFFIX)-3-g87470a2
```

Comment:
Is this related to the rest of the changes?  It isn't clear to me how.

Reply:
This includes the actual coordinate package bump.